src/components/v5/common/CompletedAction/partials/ManageReputation/ManageReputation.tsx

```diff
@@ -190,7 +195,7 @@ const ManageReputation: FC<ManageReputationProps> = ({ action }) => {
       )}
       {positiveAmountValue && (
         <>
-          {isMotion && !motionFinished ? (
+          {(isMotion || isMultiSig) && !motionFinished && !multiSigFinished ? (
```

Comment:
This logic check works, but seems a little bit prone to false negatives if certain things change in the future. I think it would be neater as:

```ts
(isMotion && !motionFinished) || (isMultiSig && !isMotionFinished)
```

But, that's just personal preference so feel free to ignore me!